src/handler/rest/auth.ts

```
@@ -1,63 +1,78 @@
-        return errorResponse(res, err)
-      })
-  }
-
-  public async register(req: Request, res: Response): Promise<void> {
-    const [err, body] = validate<RegisterRequest>(RegisterRequestSchema, req.body)
-    if (err) {
-      return errorResponse(res, err)
+    public login(req: Request, res: Response): void {
+        const [err, body] = validate<LoginRequest>(LoginRequestSchema, req.body)
+        if (err) {
+            return errorResponse(res, err)
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)